client/src/main.rs

```diff
@@ -97,6 +97,20 @@ fn main() -> std::io::Result<()> {
         "Connected to validator at: {}:{}, {}",
         args.host, args.port, ledger_info_str
     );
+    if let Some(file) = args.mnemonic_file {
+        match client_proxy.recover_wallet_accounts_by_file(&file) {
```

Comment:
I should have said this earlier, apologies, but why aren't we doing this within the new function of the client proxy. It seems weird that the ClientProxy takes this file but doesn't perform the recovery itself.

Maybe we could update ClientProxy::get_libra_wallet to call recover_wallet_accounts_by_file?

Other than this, this PR looks good :)